ubuntu/debian/libgz-msgs.install

```diff
@@ -1,4 +1,4 @@
 usr/lib/*/*.so.*
-usr/lib/ruby/gz/*.rb
```

Comment:
I think we can just revert this line and it should work

Reply:
https://github.com/gazebo-release/gz-msgs10-release/pull/3/commits/ab1399f1b8c16953a3571c312c68bdefbcb51cd6